gemrb/core/PathFinder.h

```diff
@@ -51,6 +51,32 @@ struct PathNode {
 	unsigned int orient;
 };
 
+typedef Point NavmapPoint;
+typedef Point SearchmapPoint;
+
+enum {
+	PF_SIGHT = 1,
+	PF_BACKAWAY = 2,
+	PF_ACTORS_ARE_BLOCKING = 4
+};
+
+class PQNode {
```

Comment:
PQNode is an obscure name and the class is very generic — please add a comment that we use it in our priority queue or somesuch.